org.mwc.debrief.scripting/src/org/mwc/debrief/scripting/wrappers/Tote.java

```diff
@@ -79,6 +82,9 @@ public WatchableList getPrimaryTrack()
    * Method that returns the secondary tracks of the current plot.
    * 
    * @return Array of secondary list of the current plot.
+   * 		<br />
+   *         // @type MWC.GenericData.WatchableList
```

Comment:
@helenayele - I'm pretty sure this needs array square-brackets at the end:
````
 // @type MWC.GenericData.WatchableList
````

Reply:
@IanMayo  Corrected!